draft-ietf-cdni-ci-triggers-rfc8007bis.xml

```diff
@@ -2699,10 +2739,10 @@ uCDN-A                         dCDN-B                         dCDN-C
    represent an action ("trigger") uCDN requests dCDN to carry out.</t>
 
    <t>
-   A collection of trigger resources is a resource that represents all triggers created by the dCDN at request 
+   A collection of trigger resources is a resource that represents all triggers created by the dCDN at the request 
```

Comment:
"is a resource" sounds redundant.  Maybe:
"A collection of trigger resources represents all triggers..."

Reply:
how about
"A collection of trigger resources represents all triggers created by the dCDN at the request of a uCDN, and is by itself a RESTful resource."